### DISCUSSION TITLE
setup matrix based on machine tools

### DISCUSSION THREAD

**Discussion by snelstim** [frePPLe/frepple#419] [Q&A]
Hello,

Is it possible to create a setup matrix based on tooling required for a machine? For example I have a lathe with 2 spindles, if part A and part B are both using the same size of clamping bush in the spindle then setup = 10 minutes
If part A and part B are using the same size of spindle 1 but different size of spindle 2 then setup time is 30 minutes.
If part A and part B are using 2 different spindles then setup time is 60 minutes.

How would I do this?

**Comment by jdetaeye** [comment]
The Community Edition doesn't have any setup matrix optimization capabilities.   That's a dead end.

The Enterprise Edition has extra optimization for setup matrices, but the configuration you describe isn't straightforward to model.
